### ISSUE TITLE
Known Timeline wont load 500 Error

### ISSUE THREAD

**Issue by twizzay-code** [YunoHost-Apps/pleroma_ynh#194]
Context

Build: yunohost 2.4.2

Hardware: Rpi4 4gb

Instance: [A Legendary Quest](thisis.mylegendary.quest)

[Journalctl logs](https://bin.linux.pizza/?deb8d84b845af618#HB7Lf3TRa1emuBNRMGyxPcGQsWtb6DEeVTrk76Ucipkw)

[More logs](https://bin.linux.pizza/?32aaf6b71cc1adae#BUUjx9KguWkg7xDSV2gv5f8zWKezUjvtFaA5J4ybby7J)

Using Masto FE this error does not occur however it does behave unusually slow.

I tried to install a few new FE from the Admin FE recently. After reboot, I received nginx 502 errors.

I rolled back to a previous installation and now the Known timeline is not loading and the home timeline is very slow and only loads occasionally. I am still visible to other instances. Can still post etc. but nothing from anyone else gets through the known timeline. Frequently crashes with 502 error. 

It should be noted that my current backup method is only a local backup of the pleroma installation. Not a full disk backup. I noticed after restoring from the backup that the FE’s I had installed were still there. So I manually deleted /var/lib/pleroma/static/frontends.

The problem persists. Would love to provide as much information as needed. Very motivated to solve this problem. Very little experience with pleroma. TIA

Everything worked great before I made these changes. I will shame myself and find better ways to backup my instance.

**Comment by twizzay-code** [comment]
I modified my postgresql.conf to recommendations from [PGtune](1) and opened up the timelines to be viewable from unsigned users.

After doing that, I noticed something unusual. That is, my timelines load perfectly fine for unsigned users. Its just me that is receiving the 500 internal server error for the Known and Home timelines.

Here [my instance](2) for reference. 

[1]: (https://pgtune.leopard.in.ua/)
[2]: (https://thisis.mylegendary.quest)
